### PR TITLE
Responsive desing Navbar

### PR DESCRIPTION
**Pull request by joselyepez**
Resonsive1
Responsive2

### CODE REVIEW

File: components/Navbar.jsx [hunk 1]
```diff
@@ -1,31 +1,77 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Link from "next/link";
 import Image from "next/image";
-import styles from "../styles/Navbar.module.css";
 import logo from "../public/logo.png";
+import styles from "../styles/Navbar.module.css";
+import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai";
```
Comment (Israel-Laguan) [inline components/Navbar.jsx]: I don't think using a library only for these icons is worth, please use a svg from other place instead

File: components/Navbar.jsx [hunk 2]
```diff
@@ -1,31 +1,77 @@
...
+
+const Navbar = () => {
+  const [nav, setNav] = useState(false);
```
Comment (Israel-Laguan) [inline components/Navbar.jsx]: Maybe use a name like "isNavOpen" instead

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]

File: components/Navbar.jsx [hunk 3]
```diff
@@ -2,14 +2,15 @@ import React, { useState, useEffect } from "react";
...
 import Image from "next/image";
 import logo from "../public/logo.png";
+import AiconClosed from "../public/aicons/AiconClosed.png";
+import AiconMenu from "../public/aicons/AiconMenu.png";
 import styles from "../styles/Navbar.module.css";
-import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai";
 
 const Navbar = () => {
-  const [nav, setNav] = useState(false);
+  const [nav, isNavOpen] = useState(false);
```
Comment (Israel-Laguan) [inline components/Navbar.jsx]:
```suggestion
  const [isNavOpen, setNavOpen] = useState(false);
```

Comment (Israel-Laguan) [reply components/Navbar.jsx]: Change the rest of file and we can merge

**Review by Israel-Laguan — APPROVED** [review]

**Review by elias-miguel-vergara — APPROVED** [review]

**Pull request merged**